x-pack/plugins/lens/public/visualizations/xy/xy_suggestions.ts

```diff
@@ -579,7 +579,7 @@ function buildSuggestion({
     hide:
       hide ??
       // Only advertise very clear changes when XY chart is not active
-      ((!currentState && changeType !== 'unchanged' && changeType !== 'extended') ||
+      ((!currentState && !['unchanged', 'extended', 'initial'].includes(changeType)) ||
```

Comment:
ℹ️ with this change the XY charts created by the text based suggestions are displayed correctly in Discover. I don't see any regression by doing this change instead @dej611 you can think of one.

Reply:
Reverted this change